### DISCUSSION TITLE
Clarification on Integrating New Features into Robotics Infrastructure Repository

### DISCUSSION THREAD

**Discussion by izzaaaatanishq** [JdeRobot/jderobot.github.io#264] [Q&A]
Hello everyone,

I hope this message finds you well. I've encountered a query regarding project's directory structure and the incorporation of new features.

I've observed that the robots are managed within a separate Git repository distinct from the Robotics Academy's repository. In light of this, would it be advisable for us to integrate any new features directly into the Robotics Infrastructure repository?

Furthermore, I'm curious about how these changes will be reflected in the exercises. Given that modifications will be made locally within the repository, what would be the best practice to ensure these changes are effectively synchronized with the exercises?

I appreciate any insights or guidance you can provide on this matter. Thank you for your time and assistance.

Best regards,
Tanishq

**Comment by jmplaza** [comment]
Answered at [RoboticsAcademy Discussions](https://github.com/JdeRobot/RoboticsAcademy/discussions/2437#discussioncomment-8657123).
